test/anno_verify_func_test2/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include "../../Annotations/Annotations.h"
+
+char* foo_causes_warning1() {  // warning - missing annotation MustCall("free")
+    char* x = (char*)malloc(15);
+
+    if (getchar()) {
+        return x;
+    } else {
+        return x;
+    }
+}
+
+void func1(char* x) {}
+
+char* foo_causes_warning2() { // warning - missing annotation MustCall("free"), CalledMethods("func1")
```

Comment:
I don't expect a warning about a missing `CalledMethods("func1")` annotation here, because when a caller expects an empty CalledMethods, it's okay to provide a pointer to something which has had `func1` called on it instead. More generally, for any unannotated LHS, I'd expect no CalledMethods warning - information might be lost (like the fact that `func1` really was called), but assuming nothing has been called is always safe to do.